### PR TITLE
Updated Code to Collection Lambda Function Data for use in the Graviton Opportunities Dashboard

### PR DESCRIPTION
**Pull request by erikpeteaws**
*Issue #, if available:* none

*Description of changes:*
module-inventory.yaml
Line 57: Add lambda as an Object to list
Line 843: update code for lambda

deploy-in-linked-account.yaml
Line 176: add Lambda list-function call to IAM role

module-pricing.yaml
Line 38: add Lambda to list of services

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by esc1144 — CHANGES_REQUESTED** [review]
You need to update the Mappings in the pricing module to include an entry for the AWSLambda construct. Although my deploy failed there before the inventory module, it looks like you need to create a Mappings entry for the Lambda table construct, as with the other services sub-modules

Here is the CloudFormation error in the pricing module:

> Transform AWS::LanguageExtensions failed with: Mappings not found in template for key /AWSLambda/path on resourceType Fn::ForEach::Service field forEach

**Comment by erikpeteaws** [comment]
resolving mapping issue identified with initial pull request. Mapping updated in both inventory and pricing modules for AWSLambda

File: data-collection/deploy/module-inventory.yaml [hunk 1]
```diff
@@ -789,7 +789,77 @@ Mappings:
                 paths: Arn,Name,CreatedAt,Version,accountid,collection_date,region
               SerializationLibrary: org.openx.data.jsonserde.JsonSerDe
           TableType: EXTERNAL_TABLE
-
+  AWSLambda:
```
Comment (iakov-aws) [inline data-collection/deploy/module-inventory.yaml:792]:
```suggestion
    AWSLambda:
```

**Comment by iakov-aws** [comment]
fixed a few minor things and merged here -> https://github.com/awslabs/cid-framework/pull/142

**Pull request closed without merging**